### PR TITLE
feat: add koor cluster spec and status

### PR DESCRIPTION
**Pull request by zalsader**
Includes version information

### CODE REVIEW

**Comment by galexrt** [comment]
Due to the latest changes on the `develop` branch, you must run `make gen-proto` again.

File: Makefile [hunk 1]
```diff
@@ -12,16 +13,6 @@ ifeq (, $(shell which buf))
...
 
-protoc-gen-validate: build_dir
-	if test ! -d $(BUILD_DIR)validate-$(VALIDATE_VERSION)/; then \
-		git clone --branch $(VALIDATE_VERSION) https://github.com/bufbuild/protoc-gen-validate.git $(BUILD_DIR)validate-$(VALIDATE_VERSION); \
-	else \
-		git -C $(BUILD_DIR)validate-$(VALIDATE_VERSION)/ pull --all; \
-		git -C $(BUILD_DIR)validate-$(VALIDATE_VERSION)/ checkout $(VALIDATE_VERSION); \
-	fi
-
-	cd $(BUILD_DIR) && ln -sfn validate-$(VALIDATE_VERSION)/ validate
```
[the remark this answers is not in the document]
Comment (zalsader) [reply Makefile:23]: ohh, I was wondering why it was there. I'll remove it from current pr for now and investigate later

File: pkg/k8s/koor_operator.go [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	"github.com/bufbuild/connect-go"
+	kopv1 "github.com/koor-tech/koor-operator/api/v1alpha1"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/koor-tech/data-control-center/gen/go/api/resources/stats"
+)
+
+func (k *K8s) GetKoorCluster(ctx context.Context, namespace string) (*stats.KoorCluster, error) {
+	koorClusters, err := ListCustomResource[kopv1.KoorCluster](ctx, k.client, kopv1.GroupVersion.WithResource("koorclusters"), namespace, metav1.ListOptions{})
```
Comment (galexrt) [inline pkg/k8s/koor_operator.go:16]: Please add the necessary RBAC to the helm chart `rbac.yaml`

**Review by galexrt — APPROVED** [review]
Looks good, just missing the RBAC for the koorclusters access in the helm chart

**Comment by zalsader** [comment]
I tested this locally, it works!
On to the UI
image

**Pull request merged**